CRT.c

```diff
@@ -834,7 +834,7 @@ static void dumpStderr(void) {
             fprintf(stderr, ">>>>>>>>>> stderr output >>>>>>>>>>\n");
             header = true;
          }
-         (void)! write(STDERR_FILENO, buffer, res);
+         (void)! full_write(STDERR_FILENO, buffer, res);
```

Comment:
Why keep the `!` when discarding the result anyway?

Reply:
Removed the void-cast; no longer necessary since full_write is not annotated with `__attribute__((warn_unused_result))`.